code/modules/mob/living/carbon/human/species_types/monthmen.dm

```diff
@@ -0,0 +1,102 @@
+/datum/species/monthmen
+	//an exotic species that arrives once a year to remove the worst species, mothpeople.
+	name = "Monthman"
+	id = "month"
+	//visuals
+	default_color = "FFFFFF"
+	species_traits = list(EYECOLOR,NO_UNDERWEAR,NOBLOOD,ABSTRACT_HEAD)
+	default_features = list("mcolor" = "FFF")
+	skinned_type = /obj/item/paper
+	changesource_flags = VARDROPDOWN | EVENTRACE //absolutely no way to get the race even for admins, it's completely out of theme of ss13
+	damage_overlay_type = "" //no blood
+	missing_eye_state = "montheyes_missing"
+	offset_features = list(OFFSET_UNIFORM = list(0,0), OFFSET_ID = list(0,0), OFFSET_GLOVES = list(0,0), OFFSET_GLASSES = list(0,0), OFFSET_EARS = list(0,-7), OFFSET_SHOES = list(0,0), OFFSET_S_STORE = list(0,0), OFFSET_FACEMASK = list(0,0), OFFSET_HEAD = list(0,-6), OFFSET_FACE = list(0,0), OFFSET_BELT = list(0,0), OFFSET_BACK = list(0,0), OFFSET_SUIT = list(0,0), OFFSET_NECK = list(0,0))
+	//organs
+	mutant_brain = /obj/item/organ/brain/monthmen
+	mutanteyes = /obj/item/organ/eyes/monthmen
+	mutanttongue = /obj/item/organ/tongue/monthmen
+	mutantears = /obj/item/organ/ears/monthmen
+	//other traits
+	no_equip = list(SLOT_WEAR_MASK, SLOT_GLASSES)
+	siemens_coeff = 0 //not very good at conducting electricity
+	sexes = FALSE
+	inherent_traits = list(TRAIT_NOHUNGER,TRAIT_NOBREATH)
+	meat = /obj/item/stack/sheet/cardboard
+
+/datum/species/monthmen/check_roundstart_eligible()
+	if(SSevents.holidays && SSevents.holidays[APRIL_FOOLS])
+		return TRUE
+	return ..()
+
+/datum/species/monthmen/random_name(gender,unique,lastname)
+	var/month = pick(list("January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"))
+	var/days_in_that_month = 31
+	switch(month)
+		if("February")//hey look the FUCK month where i have to calculate leap years
+			if(isLeap(text2num(time2text(world.timeofday, "YY"))))
```

Comment:
![wdada](https://user-images.githubusercontent.com/47324920/64492059-ee233b80-d26f-11e9-9fdc-6db90329d48e.PNG)

Reply:
calculating leap years in random_name was lame anyways, lets just put that on the table